src/components/Global/BaseSnackbar.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <v-snackbar
+    v-model="showSnackBar"
+    :color="color"
+    class="base-snackbar"
+    position="absolute"
+    location="bottom"
+    data-cy="auth-snackbar"
```

Comment:
Doesn't this need to read from the prop? I think we want a `data-cy` prop that this proxies down